askbot/static/askbot/css/leaderboard.scss

```diff
@@ -0,0 +1,59 @@
+.leaderboard-list .leaderboard-card {
+    border: 1px solid #ccc;
+    border-radius: $border-radius-base;
+    height: auto;
+    margin: 5px 0;
+    padding: 10px;
+
+    .leaderboard-position {
+        font-weight: bold;
+        width: auto;
```

Comment:
What do you think about adding a min-width here?  Currently the placing of the profile pictures on 9 and 10 don't line up so it looks a little off.